GitVersionCore/GitVersionFinder.cs

```diff
@@ -18,6 +18,7 @@ public SemanticVersion FindVersion(GitVersionContext context)
                 throw new Exception("NextVersion.txt has been depreciated. See https://github.com/ParticularLabs/GitVersion/wiki/GitVersionConfig.yaml-Configuration-File for replacement");
             }
 
+            //return new NewNextVersionCalculator().FindVersion(context);
```

Comment:
It turns all this new code on. When this is uncommented a whole heap of tests fail. I wanted to make sure I put the code in without affecting anything existing.

I will uncomment this on the other branch once this PR is ok